debian/changelog

```diff
@@ -688,7 +707,7 @@ ubuntu-advantage-tools (27.11~22.10.1) kinetic; urgency=medium
     - pro:
       + Ubuntu Pro is released as a product
       + make `pro` the recommended executable for the client
-      + client, apt and motd messages updated/rewritten to show Pro 
+      + client, apt and motd messages updated/rewritten to show Pro
```

Comment:
Here and above: don't change old changelog entries

Reply:
Sorry, that was my editor auto-removing trailing whitespace on save. I'll add back the trailing whitespace. Or is there any way we can correct this?